h2gis-functions/src/main/java/org/h2gis/functions/io/geojson/GeoJsonWriteDriver.java

```diff
@@ -114,7 +114,7 @@ private void writeGeoJson(ProgressVisitor progress) throws SQLException, IOExcep
                 // header of the GeoJSON file
                 jsonGenerator.writeStartObject();
                 jsonGenerator.writeStringField("type", "FeatureCollection");
-                writeCRS(jsonGenerator,SFSUtilities.getAuthorityAndSRID(connection, parse, spatialFieldNames.get(0)));
+                // writeCRS(jsonGenerator,SFSUtilities.getAuthorityAndSRID(connection, parse, spatialFieldNames.get(0)));
```

Comment:
If this line of code won't be used, you should remove it instead of commenting it.